ding/model/template/q_learning.py

```diff
@@ -67,11 +72,17 @@ def __init__(
                 action_shape,
                 layer_num=head_layer_num,
                 activation=activation,
-                norm_type=norm_type
+                norm_type=norm_type,
+                dropout=dropout
```

Comment:
add assertion in the part of `ConvEncoder`,  because you don't implement this attribute now.